lib/acts_as_taggable_on/tag.rb

```diff
@@ -89,6 +89,30 @@ def self.find_or_create_all_with_like_by_name(*list)
       end
     end
 
+    def self.update_tag(tag_name, id)
```

Comment:
@usama-ashraf can we call that `update_name`, because that's what it does.

Reply:
yes @fbrunel changes done